ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -858,34 +864,36 @@ def test_frontend_function(
                     *copy_args, array_fn=array_fn, **copy_kwargs
                 )
                 ret_ = get_frontend_ret(
-                    frontend_fn=frontend_fn,
-                    backend=backend_to_test,
+                    backend_to_test,
+                    frontend_fn,
+                    *copy_args,
                     precision_mode=test_flags.precision_mode,
                     test_compile=test_flags.test_compile,
                     frontend_array_function=(
                         create_frontend_array if test_flags.test_compile else None
                     ),
-                    *copy_args,
                     **copy_kwargs,
                 )
-                assert first_array is ret_
+                assert (
+                    first_array is ret_
+                ), f"Inplace operation failed {first_array} != {ret_}"
             else:
                 # the function provides inplace update by default
                 # check if returned reference is inputted reference
                 copy_kwargs["as_ivy_arrays"] = False
                 first_array = ivy_backend.func_wrapper._get_first_array(
-                    *args, array_fn=array_fn, **kwargs
+                    *copy_args, array_fn=array_fn, **copy_kwargs
                 )
                 ret_ = get_frontend_ret(
-                    frontend_fn=frontend_fn,
-                    backend=backend_to_test,
+                    backend_to_test,
+                    frontend_fn,
+                    *copy_args,
                     precision_mode=test_flags.precision_mode,
                     test_compile=test_flags.test_compile,
                     frontend_array_function=(
                         create_frontend_array if test_flags.test_compile else None
                     ),
-                    *args,
-                    **kwargs,
+                    **copy_kwargs,
                 )
                 assert (
                     first_array is ret_
```

Comment:
Why do we need to make changes to this file?

Reply:
The changes are needed because:

- The way **backend argument** is passed in the previous implementation causes the test to fail by raising **Value Error: got multiple values for argument backend**
- There are instances where the **test_flag.generate_frontend_arrays** will be **True** which will make the **array_fn** to be set to **_is_frontend_array**, and since the arguments in *args and **kwargs aren't frontend array types, it will cause the test to fail. So, the better arguments to use are *copy_args and **copy_kwargs which are properly modified (on line 734) based on the value of **test_flag.generate_frontend_arrays**

Reply:
@sherry30 Can you please review if these changes are needed or not? Thanks

Reply:
Hi @hmahmood24 I hope you've been good.

It's been over a week and @sherry30 hasn't given any response. What's the way forward concerning this pull request?

Reply:
@Yodeman These changes look fine to me.